website/pages/docs/plugins/sources/aws/policies.md

```diff
@@ -0,0 +1,493 @@
+# Policies and Compliance Frameworks
+
+CloudQuery SQL Policies for AWS. See the [Readme on Github](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/aws/policies) for installation instructions.
+## AWS CIS V1.2.0
+
+### Requirements
+AWS CIS V1.2.0 requires the following tables to be synced before the policy is executed:
+
+```
+tables:
+  - aws_cloudtrail_trail_event_selectors
+  - aws_cloudtrail_trails
+  - aws_ec2_flow_logs
+  - aws_ec2_security_groups
+  - aws_ec2_vpcs
+  - aws_iam_credential_reports
+  - aws_iam_password_policies
+  - aws_iam_user_access_keys
+  - aws_iam_user_attached_policies
+  - aws_iam_users
+  - aws_iam_virtual_mfa_devices
+  - aws_kms_keys
+  - aws_s3_buckets
+```
+
+### Queries
+AWS CIS V1.2.0 performs the following checks:
+  - Avoid the use of "root" account. Show used in last 30 days (Scored)
+  - Ensure MFA is enabled for all IAM users that have a console password (Scored)
+  - Ensure credentials unused for 90 days or greater are disabled (Scored)
+  - Ensure access keys are rotated every 90 days or less
+  - Ensure IAM password policy requires at least one uppercase letter
+  - Ensure IAM password policy requires at least one lowercase letter
+  - Ensure IAM password policy requires at least one symbol
+  - Ensure IAM password policy requires at least one number
+  - Ensure IAM password policy requires minimum length of 14 or greater
+  - Ensure IAM password policy prevents password reuse
+  - Ensure IAM password policy expires passwords within 90 days or less
+  - Ensure no root account access key exists (Scored)
+  - Ensure MFA is enabled for the "root" account
+  - Ensure hardware MFA is enabled for the "root" account (Scored)
+  - IAM users should not have IAM policies attached
+  - Ensure CloudTrail is enabled in all regions
+  - Ensure CloudTrail log file validation is enabled
+  - CloudTrail trails should be integrated with CloudWatch Logs
+  - Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket
+  - CloudTrail should have encryption at rest enabled
+  - Ensure rotation for customer created CMKs is enabled (Scored)
```

Comment:
```suggestion
  - Ensure rotation for customer created custom master keys is enabled (Scored)
```

Maybe it's better to use the full term instead of the acronym